README.md

```diff
@@ -1,2 +1,15 @@
-# My Lifelong Learning Lab
-Repo for code related to My Lifelong Learning Lab
+##What is My Lifelong Learning Lab?
+
+A part of the [FLOE Preference Exploration and Self-Assessment](https://wiki.fluidproject.org/display/fluid/%28Floe%29+Preference+Exploration+and+Self-Assessment) work.
+
+##Building
+
+Requirements:
+* `npm`
+
+Steps:
+* `npm install`
+    * postinstall should run the `grunt copy:frontEndDependencies` task after installation completes
```

Comment:
I wonder if there should be a cleanup task that can be used to remove the node_modules directory and the copied over frontEndDependencies for cases where someone wants to do a clean install.